site/content/docs/manifest/pipeline.md

```diff
@@ -60,6 +63,17 @@ The name of an existing CodeStar Connections connection. If omitted, Copilot wil
 
 <div class="separator"></div>
 
+<a id="build" href="#build" class="field">`build`</a> <span class="type">Map</span>  
+Configuration for CodeBuild project.
+
+<span class="parent-field">build.</span><a id="build-image" href="#build-image" class="field">`image`</a> <span class="type">String</span>  
+The URI that identifies the Docker image to use for this build project. As of now, `aws/codebuild/amazonlinux2-x86_64-standard:3.0` is used by default.
+
+!!! info
+    Copilot does not support cross-account ECR repository or private registry other than ECR because [imagePullCredentialsType](https://docs.aws.amazon.com/codebuild/latest/APIReference/API_ProjectEnvironment.html#CodeBuild-Type-ProjectEnvironment-imagePullCredentialsType) is fixed to `CODEBUILD`.
```

Comment:
What do you think of removing this information? I don't think any users asked to push images to a different registry 🤔 or different account

Reply:
This is build image used for CodeBuild project, so I thought some users might want to specify their own image stored in another account's ECR repo or another private registry. However, it maybe edge case, so I'll remove this block as you suggested. I appreciate your nice feedback!

Reply:
Thanks! let us know, once removed we can merge the PR :)

Reply:
I just pushed it🚀